profiles/pci/handhelds/profiles.toml

```diff
@@ -17,50 +17,124 @@
 #DEVICEIDS=15bf
 #83E1
 
-[steam-deck]
-desc = 'Valve Steam Deck'
+[steam-deck-jupiter]
+desc = 'Valve Steam Deck LCD'
+nonfree = false
 class_ids = "0300"
 vendor_ids = "1002"
 device_ids = "1435 163f"
-hwd_product_name_pattern = '(Galileo)|(Jupiter)'
+hwd_product_name_pattern = '(Jupiter)'
+priority = 6
+packages = 'powerbuttond jupiter-hw-support jupiter-fan-control steamdeck-dsp steam cachyos-handheld mesa lib32-mesa vulkan-radeon lib32-vulkan-radeon libva-mesa-driver lib32-libva-mesa-driver mesa-vdpau lib32-mesa-vdpau opencl-rusticl-mesa lib32-opencl-rusticl-mesa rocm-opencl-runtime'
+post_install = """
+    echo "Steam Deck Jupiter chwd installing..."
+    username=$(id -nu 1000)
+    services=("jupiter-fan-control")
+    echo "Enabling services..."
+    for service in ${services[@]}; do
+        systemctl enable --now "${service}.service"
+    done
+    #kernelparams="amd_iommu=off amdgpu.gttsize=8128 spi_amd.speed_dev=1 audit=0 iomem=relaxed amdgpu.ppfeaturemask=0xffffffff"
+    #echo "Adding required kernel parameters..."
+    #sed -i "s/LINUX_OPTIONS=\"[^\"]*/& ${kernelparams}/" /etc/sdboot-manage.conf
```

Comment:
Instead of using sed, you can create a file ``/etc/sdboot-manage.conf.d/10-chwd.conf`` and put the parameters in it:
```
LINUX_OPTIONS+=" amd_iommu=off amdgpu.gttsize=8128 spi_amd.speed_dev=1 audit=0 iomem=relaxed amdgpu.ppfeaturemask=0xffffffff"
```

Reply:
That is why the lines are commented out